invenio_cli/helpers/commands.py

```diff
@@ -182,11 +191,16 @@ def services(self, force):
                 'pipenv', 'run', 'invenio', 'index', 'queue', 'init', 'purge']
             subprocess.run(command, check=True)
 
-        # If re-run, this doesn't throw an error
+        if self.cli_config.get_services_setup():
+            return
+
+        # Without the self.cli_config.get_services_setup() check
+        # this doesn't throw an error on re-runs
         command = ['pipenv', 'run', 'invenio', 'db', 'init', 'create']
         subprocess.run(command, check=True)
 
-        # If re-run, this throws an error
+        # Without the self.cli_config.get_services_setup() check
+        # this throws an error on re-runs
         # TODO: invenio-files-rest should make it idempotent
         command = [
```

Comment:
minor: not for now, but since we will incorporate S3 this might change to a factory of location commands or so? But highly shelvable :)

Reply:
Yes, once we add support for S3 at the installation level (cookiecutter), we can change that piece of code. :+1: